overlord/ifacestate/apparmorprompting/prompting_test.go

```diff
@@ -256,6 +271,8 @@ func (s *apparmorpromptingSuite) prepManagerWithSinglePrompt(c *C, reqChan chan
 
 	path := "/home/test/foo"
 
+	whenSent := time.Now()
```

Comment:
I spent a while yesterday trying to make it easier to check new notices by putting `lastNoticeTimestamp` into `apparmorpromptingSuite` and adding some check functions akin to those in `requestrules`, but ended up throwing it away. I think this is a good simple approach. And in general this package doesn't care that much when notices are recorded, that's more a problem for `requestprompts` and `requestrules`, though it's nice to check when we can, especially when interesting synchronization is required between the two backends.